src/test/java/SalesforceModules/Compaignutil.java

```diff
@@ -0,0 +1,224 @@
+package SalesforceModules;
+
+import com.relevantcodes.extentreports.LogStatus;
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import pageobjects.SalesforceObjects.Comapign_object;
+import step_definitions.RunCukesTest;
+import utilities.HTMLReportUtil;
+import utilities.KeywordUtil;
+
+import java.util.Random;
+
+public class Compaignutil {
+
+    //P[@class='slds-truncate'][text()='Community']
+
+
+
+
+    public static void clickComapignTab(String logStep) throws InterruptedException {
+        KeywordUtil.delay(3000);
+        KeywordUtil.waitForVisible(Comapign_object.Comapign_Tab);
+        WebElement element = KeywordUtil.getDriver().findElement(Comapign_object.Comapign_Tab);
+        JavascriptExecutor executor = (JavascriptExecutor) KeywordUtil.getDriver();
+        executor.executeScript("arguments[0].click();", element);
+    }
+
+    public static void entercompaignname(String compaignname, String logstep) {
```

Comment:
Please follow proper naming convention for methods, change the method name to enterCampaignName